src/main/java/com/nvidia/spark/rapids/jni/GpuTimeZoneDB.java

```diff
@@ -27,37 +28,39 @@
 import java.util.Map;
 import java.util.TimeZone;
 import java.util.concurrent.*;
+import java.util.function.Function;
 
-import ai.rapids.cudf.ColumnVector;
-import ai.rapids.cudf.DType;
-import ai.rapids.cudf.HostColumnVector;
-import ai.rapids.cudf.Table;
+import ai.rapids.cudf.*;
 
 public class GpuTimeZoneDB {
 
   public static final int TIMEOUT_SECS = 300;
 
-
   // For the timezone database, we store the transitions in a ColumnVector that is a list of 
   // structs. The type of this column vector is:
-  //   LIST<STRUCT<utcInstant: int64, localInstant: int64, offset: int32>>
+  //   LIST<STRUCT<utcInstant: int64, localInstant: int64, offset: int32, looseInstant: int64>>
   private CompletableFuture<Map<String, Integer>> zoneIdToTableFuture;
   private CompletableFuture<HostColumnVector> fixedTransitionsFuture;
+  private CompletableFuture<HostColumnVector> zoneIdVectorFuture;
```

Comment:
What is the `zoneIdVector`? Before we had a single column described above. `zoneIdToTable` kind of was implicit, but it would be nice to add comments now that we have more columns so it is explicit, string time zone name mapped to the index into the fixedTransitions column.

Now we have `zoneIdVector`. Could you explain what is stored in it and what they are used for? Is it just a copy of `zoneIdToTable`, but to be used on the GPU? Is it sorted in any specific way?